docs/LightningNetwork.md

```diff
@@ -44,122 +45,146 @@ export BTCPAYGEN_LIGHTNING="implementationgoeshere"
 
-To ensure that your Lightning node is connected and accessible for use in your store's invoices, try to test your Lightning connection by clicking the "Public Node Info" link. Verify that a successful connection is displayed. If your internal node connection fails, make sure you have a Lightning implementation [enabled](./FAQ/LightningNetwork.md#i-previously-installed-btcpayserver-without-lightning-can-i-enable-it) on your server.
-
-If you are unable to connect to your Lightning node, try [restarting your server](./FAQ/ServerSettings.md#how-to-restart-btcpay-server) or reviewing our [troubleshooting guide](./Troubleshooting.md). You will not be able to accept lightning payments in your store until your test connection shows as successful.
+If you are unable to connect to your Lightning node, try [restarting your server](./FAQ/ServerSettings.md#how-to-restart-btcpay-server) or reviewing our [troubleshooting guide](./Troubleshooting.md). You will not be able to accept lightning payments in your store until your Lightning node appears "Online".
 
 ## Connecting an external Lightning Node in BTCPay
 
-If you would like to provide your own Lightning node connection string, select the "Use custom node" option. If you aren't sure of which settings to use, there are some examples provided on this page to help guide you. This is typically used to connect an external Lightning node to a BTCPay Server.
-
-![BTCPay Server Lightning Network Setup Custom](./img/lightning-node-setup/LightningNetworkNodeSetupCustom.jpg "BTCPay Server Lightning Network Setup Custom")
-
-## Getting Started with BTCPay and LND
+BTCPay Server offers the option to connect to an external Lightning node. To configure it:
+1. Go to "Lightning" > Select "Use custom node" if there is no Lightning node configured.
+2. Go to "Lightning" > Select "Settings" > Select "Change connection" > Select "Use custom node" to modify an existing connection
+3. Add the configuration details matching the lightning implementation used > "Test connection"
 
-The easiest way to use **LND implementation with BTCPay** is to use **[Ride The Lightning](https://github.com/ShahanaFarooqui/RTL)** (RTL).
 
-![Ride The Lightning in BTCPay Server](./img/RideTheLightning.png "Ride The Lightning in BTCPay Server")
+# Getting started with your LN implementation
+## BTCPay and LND
 
-RTL is a device agnostic **web user interface for Lightning Network** that allows you to operate your node without leaving BTCPay, from your browser.
+### Control your LND using RTL
 
-![Ride The Lightning Service in BTCPay Server](./img/server-settings/services/RideTheLightningServices.jpg "Ride The Lightning Service in BTCPay Server")
+The easiest way to use LND implementation with BTCPay is to use the **[Ride The Lightning](https://github.com/ShahanaFarooqui/RTL)** (RTL) service. A device agnostic web user interface for the Lightning Network, RTL allows you to operate your node without leaving BTCPay, from your browser. 
```

Comment:
```suggestion
The easiest way to use LND implementation with BTCPay is to use the **[Ride The Lightning](https://github.com/ShahanaFarooqui/RTL)** (RTL) service. A device agnostic web user interface for the Lightning Network, RTL allows you to operate your node without leaving BTCPay, from your browser. 
```

Wouldn't it be sufficient enough to call it a web UI for avg user ? instead of "A device agnostic web user interface"

Reply:
Sufficient indeed!